crates/viewer/re_selection_panel/src/selection_panel.rs

```diff
@@ -1322,4 +1327,156 @@ mod tests {
         harness.run();
         harness.snapshot("selection_panel_component_hybrid_overwrite");
     }
+
+    #[test]
+    fn selection_panel_view_snapshot() {
+        fn setup_blueprint(test_context: &mut TestContext) -> ViewId {
+            test_context.setup_viewport_blueprint(|_ctx, blueprint| {
+                blueprint
+                    .add_view_at_root(ViewBlueprint::new_with_root_wildcard(TestView::identifier()))
+            })
+        }
+
+        let mut test_context = get_test_context();
+        test_context.register_view_class::<TestView>();
+
+        let entity_path = EntityPath::from("static");
+        test_context.log_entity(entity_path.clone(), |builder| {
+            builder.with_archetype(
+                RowId::new(),
+                TimePoint::STATIC,
+                &MyPoints::new([MyPoint::default()]),
+            )
+        });
+
+        let view_id = setup_blueprint(&mut test_context);
+
+        // Select component:
```

Comment:
```suggestion
        // Select view:
```

or just drop the comment